src/server/dfly_bench.cc

```diff
@@ -119,6 +120,88 @@ struct ShardInfo {
 
 using ClusterSpec = vector<ShardInfo>;
 
+ClusterSpec FetchCluster(FiberSocketBase* socket, bool connection_shard_only = false) {
```

Comment:
not important now, but please take into account - we almost never use optional arguments, 
I consider them confusing when reading the code (do not quote me but I agree with Rust folks on this).